crates/inspection-service/src/inspection_service.rs

```diff
@@ -89,13 +89,20 @@ pub fn get_all_metrics() -> HashMap<String, String> {
     get_metrics(all_metric_families)
 }
 
+const CONFIGURATION_PATH: &str = "/configuration";
+const METRICS_PATH: &str = "/metrics";
+const SYSTEM_INFORMATION_PATH: &str = "/system_information";
+const JSON_METRICS_PATH: &str = "/json_metrics";
```

Comment:
nit: alphabetical? 😄

Reply:
ok, but I had to alphabetize the match cases where these are used to keep them in the same order